### PR TITLE
Pl 569

### PR DESCRIPTION
**Pull request by barkatkhan-bboxx**
1. Updated documentation of Appliance Repair Appliance Part TypeLinker 
2. Updated documentation of Appliance Part Type Replacement Linker

### CODE REVIEW

File: source/includes/_appliance_part_type_replacement_linker.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ __appliance_part_type_replacement_linker_id__ <br><font color="DarkGray">_int_</
 __<a href="/#appliance-type">appliance_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#existing-appliance-part-type">existing_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#replacement-appliance-part-type">replacement_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
+__max_part_count__  <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,default=1)</font>| Count of appliance_part_types for each Appliance
```
Comment (bryntruscott) [inline source/includes/_appliance_part_type_replacement_linker.md]: "Maximum number of parts of this type that can be replaced for this appliance type"

File: source/includes/_appliance_repair_appliance_part_type_linker.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ __<a href="/#old-appliance-part-type">old_appliance_part_type_id</a>__ <br><font
 __<a href="/#new-appliance-part-type">new_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(foreign-key)</font> | 
 __requested_date__ <br><font color="DarkGray">_datetime_</font> <font color="Crimson"></font> | 
 __replaced_date__ <br><font color="DarkGray">_datetime_</font> <font color="Crimson"></font> | 
+__repaired_parts_count__  <br><font color="DarkGray">_int_</font> | Count of parts inserted for each Appliance Repair
```
Comment (bryntruscott) [inline source/includes/_appliance_repair_appliance_part_type_linker.md]: "Number of parts of this type replaced in this repair"

**Review by bryntruscott — CHANGES_REQUESTED** [review]
Most of these files should not have changed. Only the changes to these 2 files are needed:

source/includes/_appliance_part_type_replacement_linker.md
source/includes/_appliance_repair_appliance_part_type_linker.md

**Review by bryntruscott — CHANGES_REQUESTED** [review]
Thanks for making those changes but you still need to remove all the other files from this pull request as I commented last time. Only these 2 files should be updated:

source/includes/_appliance_part_type_replacement_linker.md
source/includes/_appliance_repair_appliance_part_type_linker.md

**Comment by barkatkhan-bboxx** [comment]
It is incorrect

**Pull request closed without merging**